.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
I didn't add anything else like ruff yet as I'd like to limit the amount of code changes in here